src/Evolution/Systems/GrMhd/GhValenciaDivClean/Characteristics.hpp

```diff
@@ -0,0 +1,149 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Tags.hpp"
+#include "Evolution/Systems/GrMhd/GhValenciaDivClean/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+// IWYU pragma: no_include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+// IWYU pragma: no_include "PointwiseFunctions/Hydro/Tags.hpp"
+
+/// \cond
+class DataVector;
+namespace Tags {
+template <typename Tag>
+struct Normalized;
+}  // namespace Tags
+/// \endcond
+
+// IWYU pragma:  no_forward_declare EquationsOfState::EquationOfState
+// IWYU pragma:  no_forward_declare EquationsOfSTate::IdealFluid
+// IWYU pragma: no_forward_declare Tensor
+
+namespace grmhd {
+namespace GhValenciaDivClean {
+
+// @{
+/*!
+ * \brief Compute the characteristic speeds for the Valencia formulation of
+ * GRMHD with divergence cleaning with a dynamical spacetime evolved in the
+ * Generalized Harmonic formulation.
+ */
```

Comment:
Mention that the speeds for the combined system are just the speeds for both component systems.